bb-new.html

```diff
@@ -0,0 +1,296 @@
+              <li class="nav-list-item"><a href="#">Contact Us</a></li>
+            </ul>
+          </div>    
+        </div>                 
+        </nav>                 
+            <div class="row">
+              <div class="col">
+                <h1>BlackBerry 2021 Threat Report Now Available</h1>
+                <p>Learn about the most significant cybersecurity events of 2020 so you can prepare for the year ahead</p>
+                <button>READ THE REPORT</button>
+              </div>
+            </div>
+          </header>         
+        </div>         
+      </div>
+      <div class="row">
+        <div class="col">
+          <h2>SECURITY SHOULD'T JUST BE SMART. IT SHOULD BE INTELLIGENT</h2>
+          <p>We’re known for secure handsets that changed the world, and how we work. Today, we’re reimagining protection, again, with artificial intelligence and machine learning. We make everything more secure—and you, more productive.</p>
+          <ul>
+            <li class="sec-list-items"><a href="#">CYBERSECURITY</a></li>
+            <li class="sec-list-items"><a href="#">CRITICAL EVENT MANAGEMENT</a></li>
+            <li class="sec-list-items"><a href="#">INTERNET OF THINGS (loT)</a></li>
+          </ul>
+        </div>
+        <div class="col">
+          <img src="/images/men-holding-laptop-on-baconey.jpg" alt="men holding laptop outside on a baconey">
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <img src="/images/men-back-sitting-on-chair.jpg" alt="men back sitting on a chair facing a laptop">
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <img src="/images/security-icon.jpg" alt="blackberry security icon">
+          <h2>WE’VE NEVER BEEN MORE CONNECTED.OR MORE VULNERABLE.</h2>
+          <p>Cybersecurity has failed to keep up, because it fails to look ahead. Our intelligent security pairs artificial intelligence with machine learning to proactively protect your system from cyberthreats. It’s time to protect, prevent and respond.</p>
+          <button syle="display: inline-block">CYBER SUITE</button>
+          <button style="display: inline block">SPARK SUITE</button>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <img src="/images/security-icon-2.jpg" alt="blackberry security icon">
+          <h2>ALWAYS-ON. ALWAYS PROTECTED</h2>
+          <p>In times of crisis, mobility and secure communications are essential. Our critical event management solutions are built on our trusted reputation of absolute reliability. We don’t just keep communications safe—we keep people safe.</p>
+          <button syle="display: inline-block">BLACKBERRY ATHOC</button>
+          <button style="display: inline block">BLACKBERRY ALERT</button>
+        </div>
+      </div>
+      <div class="row">
+        <div class="col">
+          <img src="/images/astronaut.jpg" alt="astronaut floting">
```

Comment:
Again, why is this in its own row, separate from the "For where you are now..." content?  This needs to be beside it, which means it needs to be in the same row, but in a different column